### PR TITLE
Update install-minikube.md

### PR DESCRIPTION
**Pull request by shavidissa**
Added steps to find out if virtualization in installed on the users OS.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Hi, @shavidissa. I think this PR is already in process, https://github.com/kubernetes/website/pull/14013.

**Comment by zacharysarah** [comment]
@shavidissa Is this to close a specific issue?

**Comment by shavidissa** [comment]
Hi @zacharysarah: No I noticed this gap and wanted to update docs but it looks like there is a PR already on the way for it :) I will close this PR.

Hi @kbhawkey: Thank you for bringing it to my notice. :)

**Comment by shavidissa** [comment]
Ok @zacharysarah  and @kbhawkey am reopening this PR as am not sure if the mentioned PR addresses this issue. What do you think?

**Comment by zacharysarah** [comment]
@shavidissa It looks #14013 has a different scope, but it's also proposing changes to some of [the same content](https://github.com/kubernetes/website/pull/14013/files#diff-f313eb2e4ec2040fea6586c1a2477801R18). 

@sftim 👋 Any input here?

**Comment by sftim** [comment]
I'm happy to collaborate. Might not have many cycles this week though.

**Comment by sftim** [comment]
I can rebase if this gets merged first.

File: content/en/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -15,10 +15,37 @@ This page shows you how to install [Minikube](/docs/tutorials/hello-minikube), a
...
+```
+If you see `VMX` in the output, the VT-x feature is supported on your OS.
+{{% /tab %}}
+{{% tab name="Windows" %}}
+To check if virtualization is supported on Windows 8 and above, run the following command on your Windows terminal or command prompt. 
+```shell
+systeminfo
+```
+If you see the following output, virtualization is supported on Windows.
+```shell
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-minikube.md]: This is output, so I would not mark it as shell syntax.

**Comment by shavidissa** [comment]
Hi @zacharysarah, I squashed the commits to 1.  :)
Hi @sftim, I removed the shell syntax. Thanks a lot for the support :)

**Comment by tengqm** [comment]
/lgtm

The preview looks nice.

**Pull request merged**